packages/back-end/src/models/FeatureRevisionModel.ts

```diff
@@ -161,6 +163,7 @@ export async function getFeatureRevisionsByStatus({
   })
     .select("-log") // Remove the log when fetching all revisions since it can be large to send over the network
     .sort({ version: -1 })
```

Comment:
Sorting by newest first makes the offset unstable.  If a new version is added while your script is running, you could end up fetching the same revision twice.  Newest first makes sense for the other places that call this method, so maybe for the new REST endpoint only, we do oldest first.